doc/user/content/integrations/redpanda-cloud.md

```diff
@@ -0,0 +1,100 @@
+
+    c. Click on the **Security** tab
+
+    d. In the **Security** section, choose **Add service account**
+
+    e. Specify a name for the service account and click **Create**
+
+    Take note of the name of the service account you just created, as well as the service account secret key; you'll need them later on. Keep in mind that the service account secret key contains sensitive information, and you should store it somewhere safe!
+
+    ##### Create an Access Control List (ACL)
+    a. Next, while you're at the **Security** section, click on the **ACL** tab
+
+    b. Click **Add ACL**
+
+    c. Choose the **Service Account** that you've just created
+
+    d. Choose the **Read** and **Write** permissions for the **Topics** section
+
+    e. You can grant access on a per-topic basis, but you can also choose to grant access to all topics by adding a `*` in the **Topics ID** input field.
+
+    f. Click on **Create**
+
+3. #### Download the Redpanda Broker CA certificate
+    a. Go to the [Redpanda Cloud console](https://vectorized.cloud/)
+
+    b. Click on the **Overview** tab
+
+    c. Click on the **Download** button next to the **Kafka API TLS**
+
+    d. You'll need to store it somewhere safe; when creating the Redpanda source in Materialize you'll need to provide the absolute path to the certificate authority (CA) certificate file in order to securely connect Redpanda Cloud to Materialize.
+
+4. #### Create a topic
+    To start using Materialize with Redpanda, you need to point it to an existing Redpanda topic you want to read data from.
+
+    If you already have a topic created, you can skip this step.
+
+    Otherwise, you can install Redpanda on your client machine from the previous step and create a topic. You can find more information about how to do that [here](https://docs.redpanda.com/docs/reference/rpk-commands/#rpk-topic-create).
+
+
+5. #### Create a source in Materialize
+    a. Open the [Redpanda Cloud console](https://vectorized.cloud/) and select your cluster
+
+    b. Click on **Overview**
+
+    c. Copy the URL under **Cluster hosts**. This will be your `<broker-url>` going forward
+
+    d. From the _psql_ terminal, run the following command. Replace `<redpanda_cloud>` with whatever you want to name your source. The broker URL is what you copied in step c of this subsection. The `<topic-name>` is the name of the topic you created in Step 4. The `<your-username>` and `<your-password>` are from the _Create a Service Account_ step.
+    ```sql
+    CREATE SOURCE <redpanda_cloud>
+      FROM KAFKA BROKER '<redpanda-broker-url>' TOPIC '<topic-name>' WITH (
+          sasl_mechanisms = 'SCRAM-SHA-256',
+          security_protocol = 'SASL_SSL',
+          sasl_username = '<your-username>',
+          sasl_password = '<your-password>',
+          ssl_ca_location = '/secrets/ca.crt'
+      )
+      FORMAT BYTES;
+    CREATE MATERIALIZED VIEW <topic_name> AS
+      SELECT * FROM <redpanda_cloud>;
```

Comment:
```suggestion
```

Reply:
This is missing the intermediate conversion view step, so might as well just cut it out!